components-core/src/main/java/com/adyen/checkout/components/util/ValidationUtils.kt

```diff
@@ -1,11 +1,15 @@
+@file:Suppress("MaximumLineLength", "MaxLineLength")
```

Comment:
it doesn't work for property and class.

Reply:
The problem seems to be coming from `MaximumLineLength` only. Would it work if we move `MaxLineLength` from here into the `Suppress` on top of `EMAIL_REGEX`? I'm trying to avoid disabling a rule for the whole file.

Reply:
Yes, it works. I changed there